Ingress/controllers/nginx-third-party/nginx/manager.go

```diff
@@ -0,0 +1,168 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package nginx
+
+import (
+	"reflect"
+
+	"k8s.io/contrib/Ingress/controllers/nginx-third-party/ssl"
+
+	"github.com/fatih/structs"
+	"github.com/golang/glog"
+
+	"k8s.io/kubernetes/pkg/apis/extensions"
+	"k8s.io/kubernetes/pkg/client/unversioned"
+	"k8s.io/kubernetes/pkg/util"
+)
+
+const (
```

Comment:
please add a comment for each of these kind of like we do in the serviceloadbalancer (ideally a brief explanation followed by some pro/con of increasing it: https://github.com/kubernetes/contrib/blob/master/service-loadbalancer/service_loadbalancer.go#L58). Use your judgement to ignore the obvious ones (at least defIPCIDR is non-obvious).